<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurazioneDlg</class>
 <widget class="QDialog" name="ConfigurazioneDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generaleTab">
      <attribute name="title">
       <string>Generale</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nome Cassa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nomeCassaTxt">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Visualizzazione automatica del resto</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="attivaRestoCheck">
            <property name="text">
             <string>Attiva</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="durataRestoLbl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Secondi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="durataRestoTxt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>durataRestoTxt</zorder>
         <zorder>layoutWidget</zorder>
         <zorder>attivaRestoCheck</zorder>
         <zorder>horizontalSpacer</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Stampa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Stampante</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="stampanteSelezionataTxt"/>
         </item>
         <item>
          <widget class="QToolButton" name="printerSelectBtn">
           <property name="text">
            <string>Cambia</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Intestazione 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="intestazione1Txt"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Intestazione 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="intestazione2Txt"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Intestazione 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="intestazione3Txt"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Database</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nomeCassaTxt</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>stampanteSelezionataTxt</tabstop>
  <tabstop>printerSelectBtn</tabstop>
  <tabstop>intestazione1Txt</tabstop>
  <tabstop>intestazione2Txt</tabstop>
  <tabstop>intestazione3Txt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
